package.json

```diff
@@ -22,6 +22,7 @@
     "standard": "^9.0.2",
     "truffle": "^3.2.1",
     "truffle-contract": "^2.0.0",
+    "uport-connect": "^0.5.6",
```

Comment:
@christianlundkvist We should update this to 0.6.0 the latest version so it supports MNID's